google/auth/external_account.py

```diff
@@ -85,6 +85,7 @@ def __init__(
         default_scopes=None,
         workforce_pool_user_project=None,
         universe_domain=_DEFAULT_UNIVERSE_DOMAIN,
+        trust_boundary=None,
```

Comment:
Why expose this constructor option right now if we aren't using it anywhere?

Reply:
True. It can be either now or later when we really do the logic.